README.md

```diff
@@ -116,10 +116,10 @@ these values should be.
 
 If you see `Serving requests at 0.0.0.0:8080` the dev server is working.
 
-You may need to [create a service account in the GCP
-dashboard](https://pantheon.corp.google.com/iam-admin/serviceaccounts?project=flutter-dashboard&supportedpurview=project)
-to develop and test some features; these cannot be tested if you do
-not work for Google currently.
+To develop and test some features, you need to have a local `key.json` 
+copy of a service account ([dart-cocoon](https://pantheon.corp.google.com/iam-admin/serviceaccounts/details/110857733037195084916?project=flutter-dashboard)) in the GCP dashboard. Please refer to the 
```

Comment:
To develop and test some features, you need to have a local service account(`key.json`)
with access to the project you will be connecting to.

If you work for Google you can use the ... key with flutter-dashboard project.

Reply:
updated.